cypress/page/menu-content.page.ts

```diff
@@ -5,7 +5,7 @@ class MenuContentPage {
 
 
   constructor() {
-    this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
+    this.tShirtMenu = "[title='T-shirts']";
```

Comment:
as we saw it in the last class, the appropriate selector should be:
`#block_top_menu > ul > li > a[title='T-shirts']`

So you don't need the call to the `eq` command if you use this selector

Reply:
Thanks holmes